### PR TITLE
#1 Create contact form

### LINKED ISSUE #1: Create contact form
Create contact form for blogueras techie app

fields:
- [ ] name
- [ ] email
- [ ]  message

### PR DESCRIPTION
**Pull request by dgattoni**
closes #1 

Added contact form.

### CODE REVIEW

**Comment by dgattoni** [comment]
it's ready for review 

cc: @meligatt

File: app/controllers/contact/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = function(req, res) {
   
-  var title = 'contact form';
+  var title = 'Blogueras techie contact form';
```
Comment (meligatt) [inline app/controllers/contact/index.js]: i dont like the name of the variable ;p

Comment (dgattoni) [inline app/controllers/contact/index.js]: oka!aaaaaaaa

**Comment by meligatt** [comment]
@dgattoni  ok, i like the new changes, i'll merge. good job

**Pull request merged**